### DISCUSSION TITLE
Use of hyper links in Elixir

### DISCUSSION THREAD

**Discussion by fdRosino** [facebook/docusaurus#9962] [Q&A]
Hi,
how can I use **hyper link** in Mind Elixir, I can't find any news,
Thank you

**Comment by Advait251206** [comment]
Mind Elixir doesn't currently have built-in support for hyperlinks on nodes (e.g. clickable URLs rendered directly as part of the node content).

If your goal is to associate a URL with a node, here are a few options:

- Store the URL as part of the node's data and handle it yourself in your application.
- Listen for node click or selection events, then open the associated URL using `window.open()` or your preferred navigation logic.
- Use custom node rendering (if it fits your use case) to display clickable content.

If you're looking for native hyperlink support (similar to what some other mind mapping tools provide), I don't believe that's currently available. In that case, it would be worth opening a feature request if one doesn't already exist.

If you can share your use case (for example, opening external websites, linking to internal pages, or linking between nodes), the community may be able to suggest the most appropriate approach.

--

If this answer helped or pointed you in the right direction, I'd appreciate it if you could mark it as the accepted answer so it's easier for others with the same issue to find.

Also, if you found my contribution useful, I'd appreciate it if you could check out my GitHub profile, follow me, and star any repositories you find interesting.

GitHub: https://github.com/Advait251206
